### DISCUSSION TITLE
Outputting symbols to local directory

### DISCUSSION THREAD

**Discussion by 1iveowl** [nektos/act#1604] [Q&A] [answered]
Is is possible to run the workflow and then output specific files (specifically .pdb) to a local directory?

Scenario:
Using Act in an inner dev-test loop to deploy an web app build (with configuration set to debug) to Azure App Services, and then want to attach a remote debugger. To do this the pdb symbole files are needed. I can download them manually using Kuda, but looking to optimize by having the workflow run by act output these files to a local directory.

I guess the solution could be related to attaching a Docker volume and directing output to this volume. Maybe specifying a `cp` command in the workflow yaml file.

**Comment by catthehacker** [comment]
you can use artifact server for that

**Comment by 1iveowl** [comment, marked as answer]
Thanks for pointing me in the right direction will look in to this. 

Looking at the options, is it sufficient to provide a local path or do I need to spin up a local server (i.e., in another container) that will manage the output by act? 

Would be great if there was some example of how to actually use this feature. 

Thanks
